ocaml/xapi/xapi_vm_helpers.ml

```diff
@@ -117,6 +117,8 @@ let set_is_a_template ~__context ~self ~value =
          )
       |> List.rev
       |> List.iter (fun p -> Db.PVS_proxy.destroy ~__context ~self:p) ;
+      (* Remove the VM from all groups *)
```

Comment:
Update the comments as the below one to avoid confusion, like:
"(* delete the associated vms if any exists, when we templat'ize it *)"

Reply:
Why `templat'ize` not `templatize`?

Reply:
OK, maybe it is not right, I copied from the below comment.